provision-contest/ansible/roles/mysql_replication/tasks/main.yml

```diff
@@ -1,6 +1,13 @@
 ---
 # These tasks configure a MySQL server for master/master replication
 
+- name: Install master dependency for ipaddr jinja filter
```

Comment:
I’m not sure if I like this. I sometimes run ansible from my Mac and then this wouldn’t work?

Reply:
I can try with https://docs.ansible.com/ansible/latest/collections/ansible/builtin/package_module.html but I don't think homebrew will understand. Otherwise I have to check the OS of the control node in a different play.

Reply:
@nickygerritsen I've now added another play and make target (https://github.com/DOMjudge/domjudge-scripts/pull/83/commits/39caf4b42e1ba1e8f33f2078faebc30ba29ec691) which basically does a pretask to install the needed dependencies.